assets/scss/print.scss

```diff
@@ -31,6 +31,18 @@
   margin-top: 8.5mm;
 }
 
+.app-u-print-full-width {
+  width: 100%;
```

Comment:
Does this need a `!important` or similar?
Just wondering which will take precedence when used with other width based classes?
```
<div class="govuk-grid-column-two-thirds app-u-print-full-width">
```

Reply:
Usually with a utility class you probably would use `!important` but in this case in print, the `print.css` file is taking precedence and `app-u-print-full-width` has higher specificity than `govuk-grid-column-two-thirds`

Reply:
I cant claim to fully understand how that works, but 🤷‍♂️ , if you are happy then I'm happy 😁